src/main/resources/db/migration/V1_13__update_prison_names.sql

```diff
@@ -0,0 +1,43 @@
+UPDATE PRISON SET NAME = 'Askham Grange (HMP/YOI)' WHERE PRISON_ID = 'AGI';
```

Comment:
Updating `(HMP & YOI)` to `(HMP/YOI)` as per spreadsheet which brings us to be consistent with GOV.UK prison pages